lib/CourseThemeModule.js

```diff
@@ -132,64 +132,50 @@ class CourseThemeModule extends AbstractApiModule {
    */
   async processFileVariables (fwBuild) {
     const assets = await this.app.waitForModule('assets')
-    const fontDest = `${fwBuild.dir}/src/theme/${fwBuild.courseData.config.data._theme}/fonts`
+    const themeDir = `${fwBuild.dir}/src/theme/${fwBuild.courseData.config.data._theme}`
     const fontData = fwBuild.courseData.course.data.themeVariables._font
-    let fontImportCSS = ''
 
-    const fontWeightCheck = {
-      'woff2-light': function () {
-        return 'font-weight: light;\n\tfont-style: normal;'
-      },
-      'woff2-lightItalic': function () {
-        return 'font-weight: light;\n\tfont-style: italic;'
-      },
-      'woff2-normal': function () {
-        return 'font-weight: normal;\n\tfont-style: normal;'
-      },
-      'woff2-normalItalic': function () {
-        return 'font-weight: normal;\n\tfont-style: italic;'
-      },
-      'woff2-bold': function () {
-        return 'font-weight: bold;\n\tfont-style: normal;'
-      },
-      'woff2-boldItalic': function () {
-        return 'font-weight: bold;\n\tfont-style: italic;'
-      }
+    if (!fontData) {
+      return ''
     }
-    // Font files
-    if (fontData._items !== null) {
+    let fontImportCSS = ''
+    // font files
+    if (fontData._items) {
```

Comment:
Removed all null comparisons, as these won't work when accessing object properties (should be undefined, or falsy)